internal/scheduler/database/job_repository.go

```diff
@@ -73,6 +79,71 @@ func NewPostgresJobRepository(db *pgxpool.Pool, batchSize int32) *PostgresJobRep
 	}
 }
 
+func (r *PostgresJobRepository) FetchInitialJobs(ctx *armadacontext.Context) ([]Job, []Run, error) {
+	var updatedJobs []Job = nil
```

Comment:
```suggestion
	var updatedJobs []Job
```

Default value is `nil`

Reply:
Most of this is copy pasted from the existing `FetchJobUpdates` code. Do you want me to fix that code too?

Reply:
I guess we can fix it